app/controllers/solutions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SolutionsController < ApplicationController
+
+  def create
+    @solution = Solution.new(solution_params)
+    @question = @solution.question
```

Comment:
方向性はOKです。何しろ`@question`が必要になることを踏まえると，次のように実装するのが効率良いです。

```suggestion
    @question = Question.find(params[:question_id])
    @solution = @question.solutions.build(solution_params)
```

なお，`solution_params` から`question_id` は削りましょう。